database/things/readable-lisp.pldb

```diff
@@ -1,4 +1,5 @@
 title readable-lisp
+appeared 1958
```

Comment:
year?

Reply:
Hi @breck7 , I will be sending new PR including more appeared year coverage. Will update it in that. Is it fine?